WTWSMS/events/WtWSMS_story_events.txt

```diff
@@ -3599,4 +3598,112 @@ narrative_event = {
 	option = { 
 		name = historical_wtwsms.6.a
 	}
-} 
\ No newline at end of file
+}
+
+# Emperor has won the Isaurian Wars
+narrative_event = {
+	id = historical_wtwsms.7
+	title = historical_wtwsms.7.name
+	desc = historical_wtwsms.7.desc
+	picture = GFX_evt_victory_arch_byzantine
+	
+	only_rulers = yes
+	only_independent = yes
+	
+	trigger = {
+		start_date = 492.1.1
+		NOT = { year = 520 } # the war should be ended by this year
+		ROOT = {
+			has_landed_title = e_byzantium
+			OR = {
+				has_landed_title = c_ephesos # Capital of Asia
+				has_landed_title = c_ikonion # Capital of Pisidia
+				any_realm_lord = {
+					has_landed_title = c_ephesos # Capital of Asia
+				}
+				any_realm_lord = {
+					has_landed_title = c_ikonion # Capital of Pisidia
+				}
+			}
+		}
+		d_isauria = {
+			holder_scope = {
+				has_truce = ROOT
+			}
+		}
+		NOT = { has_global_flag = isaurian_war_ended }
+	}
+	
+	mean_time_to_happen = {
+		days = 1
+	}
+	
+	immediate = {
+		set_global_flag = isaurian_war_ended
+		d_isauria = { holder_scope = { save_global_event_target_as = target_isaurian_leader } }
+	}
+	
+	option = {
+		name = OK
+		
+		prestige = 100
+		d_isauria = {
+			holder_scope = {
+				abdicate_to = ROOT
+				imprison = ROOT
+			}
+		}
+		clear_global_event_target = target_isaurian_leader
+	}
+}
+
+# Isaurians have won the Isaurian Wars
+narrative_event = {
+	id = historical_wtwsms.8
+	title = historical_wtwsms.8.name
+	desc = historical_wtwsms.8.desc
+	picture = GFX_evt_castle_hill
+	
+	only_rulers = yes
+	only_independent = yes
+	
+	trigger = {
+		start_date = 492.1.1
+		NOT = { year = 520 } # the war should be ended by this year
+		ROOT = {
+			has_landed_title = d_isauria
+			OR = {
+				has_landed_title = c_ephesos # Capital of Asia
+				has_landed_title = c_ikonion # Capital of Pisidia
+				any_realm_lord = {
+					has_landed_title = c_ephesos # Capital of Asia
+				}
+				any_realm_lord = {
+					has_landed_title = c_ikonion # Capital of Pisidia
+				}
+			}
+		}
+		e_byzantium = {
+			holder_scope = {
+				has_truce = ROOT
+			}
+		}
+		NOT = { has_global_flag = isaurian_war_ended }
+	}
+	
+	mean_time_to_happen = {
+		days = 1
+	}
+	
+	immediate = {
+		set_global_flag = isaurian_war_ended
+		d_isauria = { holder_scope = { save_global_event_target_as = target_isaurian_leader } }
```

Comment:
Same as above, the one receiving the event is the holder of d_isauria.

Reply:
OK, I will simplyfy this

Reply:
Done.

Reply:
You don't need this at all, actually. It isn't used in the localisation for that event,

Reply:
But you were talking about expanding this event's localisation, so I wanted to make it future-proof

Reply:
I already did that. And again, Root will always be used.

Reply:
Done, removed.